### ISSUE TITLE
search bar doesn't focus automatically

### ISSUE THREAD

**Issue by Ali-Hussein-dev** [quran/quran-ios#390]
When taping on the search icon on the bottom bar, it doesn't focus automatically. I am not sure if it is intentional, but that is an expected behavior to happen.

**Comment by mohamede1945** [comment]
Yes, this is expected. So one can choose one of the recent search terms. I'll probably expand the limit of the recent search terms to allow more terms there.

**Comment by Ali-Hussein-dev** [comment]
Maybe I haven't explained it well! Currently, users have to tap twice to focus on the search bar, first tap on the search icon and second tap on the search bar. What I think should solve it, is to focus the search bar by just tapping on the search icon. hope that is useful.

**Comment by mohamede1945** [comment]
Yes, understood. This is very similar experience to App Store for example.

**Comment by Ali-Hussein-dev** [comment]
yes and I don't know how it makes sense!

**Comment by mohamede1945** [comment]
I think to allow the users to see the other data in the screen. For App store, they have suggestions there. For this Quran app, we have recent search terms. If the keyboard is up, the user won't see the recent search terms.

**Comment by mohamede1945** [comment]
I get your point though. I also see App store's point. I'm just not sure which way is better.

**Comment by Ali-Hussein-dev** [comment]
I think the better would be the one which mostly occurs.

**Comment by amerbouh** [comment]
@Ali-Hussein-dev I totally get the point you're making, but I personally think its best to keep the current behaviour, since this is what users in the Apple Ecosystem are used to. Here are the Search guidelines provided by Apple, which don't precise whether or not the search bar should be displayed automatically: https://developer.apple.com/design/human-interface-guidelines/patterns/searching. Again, I think we should stick to what is common on the platform we're designing for.
